vtds_cluster_kvm/private/config/config.yaml

```diff
@@ -254,14 +250,47 @@ cluster:
         # The 'boot_disk' block describes the characteristics of the
         # boot disk, including the OS image it should be created from.
         boot_disk:
-          # The 'source_image' field specifies the URL of the OS QCOW
-          # image from which the boot disk for each Virtual Node of
-          # this class should be built.
-          source_image: "http://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img"
+          # The 'source_image' field specifies the URL of the OS or
+          # installation media image from which the boot disk for each
+          # Virtual Node of this class should be built.
+          source_image: http://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img
```

Comment:
REVIEWERS: I have added base configuration support for checking the source checksum for the boot disk image. I have not implemented the check yet, but this will be needed fairly soon, and I was in here making other changes, so I am laying the groundwork for that now.